### PR TITLE
[Fix]: Table graph select column issue

### PR DESCRIPTION
**Pull request by nxanil**
@shashank-nx

Fixed issue https://github.com/nuagenetworks/vis-graphs/issues/463. Please verify once.

@preshitg  

Please review the changes.

### CODE REVIEW

**Comment by shashank-nx** [comment]
@preshitg @nxanil 

it's working fine.

**Comment by preshitg** [comment]
@nxanil @shashank-nx What is this issue? Please describe either in this PR or in the github issue.

File: Graphs/Table/index.js [hunk 1]
```diff
@@ -175,7 +176,7 @@ const TableGraph = (props) => {
             updateScroll,
         } = props;
 
-        if (!isEmpty(displayColumn)) {
+        if (!isEmpty(displayColumn) || isAllColumnSelected === true) {
```
Comment (preshitg) [inline Graphs/Table/index.js:179]: `!isEmpty(displayColumn) || isAllColumnSelected` should suffice since isAllColumnSelected always has a boolean value

**Comment by nxanil** [comment]
> @nxanil @shashank-nx What is this issue? Please describe either in this PR or in the github issue.

@preshitg 
In Table graph, when selecting all the column to be display from selectColumn option and refreshing the table graph, it was reinitialized to show default selected column only.
selectColumn

**Review by preshitg — APPROVED** [review]

**Pull request merged**